lib/msf/ui/debug.rb

```diff
@@ -0,0 +1,208 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Displays Metasploit information useful for Debugging.
+    #
+    ###
+    module Debug
+      COMMAND_HISTORY_TOTAL = 50
+      ERROR_TOTAL = 10
+      LOG_LINE_TOTAL = 50
+
+      def self.get_issue_link
+        return 'https://github.com/rapid7/metasploit-framework/issues/new'
+      end
+
+      def self.get_preamble
+        return ("\nPlease provide the below information in any Github issues you open. New issues can be opened here #{get_issue_link}\n"+
+          "%red%undENSURE YOU HAVE REMOVED ANY SENSITIVE INFORMATION BEFORE SUBMITTING!%clr\n\n")
```

Comment:
Heredocs would help clean this up:

```suggestion
<<~PREMABLE
  Please provide the below information in any Github issues you open. New issues can be opened here #{get_issue_link}
  %red%undENSURE YOU HAVE REMOVED ANY SENSITIVE INFORMATION BEFORE SUBMITTING!%clr
  
   ===8<=== CUT AND PASTE EVERYTHING BELOW THIS LINE ===8<===

PREMABLE
```